src/wp-admin/authorize-application.php

```diff
@@ -18,6 +18,7 @@
 	$success_url = $_POST['success_url'];
 	$reject_url  = $_POST['reject_url'];
 	$app_name    = $_POST['app_name'];
+	$app_id      = ! empty( $_REQUEST['app_id'] ) ? $_REQUEST['app_id'] : '';
```

Comment:
Should this be `$_POST` ?  Is there any reason we'd accept from either COOKIE or GET or the like?

Reply:
Also, as this is handling the form we build down below, is there any circumstance where it wouldn't be set?  I think we can just do `$app_id      = $_POST['app_id'];` instead -- possibly with a `(int)`

Reply:
Good point, we can simplify to `$_POST['app_id']`.